src/components/logicComponents/Carousel.js

```diff
@@ -9,8 +9,8 @@ export function Carousel({ movies }) {
   const scrollRef = useHorizontalScroll();
   return (
     <div ref={scrollRef} className="Carousel-container">
-      {movies !== undefined && movies.length !== 0 ? (
-        movies?.map(movie => {
+      {!!movies?.length ? (
```

Comment:
Let's avoid the conditional. Even if you're not adding the `if` statement, that's what's happening under the hood. Also there is not need for that all the time.

```suggestion
      {!!movies.length ? (
```